pkg/api/extensions/accessor.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Nice! But shouldn't this rather go to `pkg/apis/extensions`? Why creating a new `pkg/api/extensions` package now?

Reply:
You'll have cyclic imports otherwise :/ I used the same pattern as in `meta.Accessor` of k8s.

Reply:
ok, thanks!